eng/cibuild_bootstrapped_msbuild.ps1

```diff
@@ -106,6 +106,11 @@ try {
 
   if ($buildStage1)
   {
+    if (Test-Path $Stage1Dir)
```

Comment:
I think you included code from outside your pr. This is an example from Mihai. Would you mind cleaning it up before I look further?